src/blog/v8-release-97.md

```diff
@@ -0,0 +1,26 @@
+---
+title: 'V8 release v9.7'
+author: 'Ingvar Stepanyan ([@RReverser](https://twitter.com/RReverser))'
+avatars:
+ - 'ingvar-stepanyan'
+date: 2021-11-05
+tags:
+ - release
+description: 'V8 release v9.7 brings new JavaScript methods for searching backwards in arrays.'
+tweet: ''
+---
+Every four weeks, we create a new branch of V8 as part of our [release process](https://v8.dev/docs/release-process). Each version is branched from V8’s Git main immediately before a Chrome Beta milestone. Today we’re pleased to announce our newest branch, [V8 version 9.7](https://chromium.googlesource.com/v8/v8.git/+log/branch-heads/9.7), which is in beta until its release in coordination with Chrome 97 Stable in several weeks. V8 v9.7 is filled with all sorts of developer-facing goodies. This post provides a preview of some of the highlights in anticipation of the release.
+
+## JavaScript
+
+### `findLast` and `findLastIndex` on Array and TypedArray prototypes
+
+The `findLast` and `findLastIndex` methods find elements that match a predicate from the end of an Array. For example, `[1,2,3,4].findLast((el) => el %2 === 0)` returns the last even element, `4`. These methods are available without a flag starting in v9.7.
```

Comment:
Small block is usually a bit more readable for primary code examples (plus adding a missing space after `%`).

````suggestion
The `findLast` and `findLastIndex` methods find elements that match a predicate from the end of an Array. For example:

```js
[1,2,3,4].findLast((el) => el % 2 === 0)
// → 4 (last even element)
```

These methods are available without a flag starting in v9.7.
````